lib/CANNON-shape2mesh.js

```diff
@@ -89,39 +100,39 @@ CANNON.shape2mesh = function(body){
                         v0.vadd(shape.pillarOffset, v0);
                         v1.vadd(shape.pillarOffset, v1);
                         v2.vadd(shape.pillarOffset, v2);
-                        geometry.vertices.push(
+                        vertices.push(
                             new THREE.Vector3(v0.x, v0.y, v0.z),
                             new THREE.Vector3(v1.x, v1.y, v1.z),
                             new THREE.Vector3(v2.x, v2.y, v2.z)
                         );
-                        var i = geometry.vertices.length - 3;
-                        geometry.faces.push(new THREE.Face3(i, i+1, i+2));
+                        var i = vertices.length - 3;
+                        faces.push(i, i+1, i+2);
                     }
                 }
             }
-            geometry.computeBoundingSphere();
-            geometry.computeFaceNormals();
+            var geometry = createBufferGeometry(vertices, faces);
             mesh = new THREE.Mesh(geometry, this.currentMaterial);
             break;
 
         case CANNON.Shape.types.TRIMESH:
-            var geometry = new THREE.Geometry();
+            var geometry = new THREE.BufferGeometry();
 
             var v0 = new CANNON.Vec3();
             var v1 = new CANNON.Vec3();
             var v2 = new CANNON.Vec3();
+            var vertices = [];
+            var faces = [];
             for (var i = 0; i < shape.indices.length / 3; i++) {
                 shape.getTriangleVertices(i, v0, v1, v2);
-                geometry.vertices.push(
+                vertices.push(
                     new THREE.Vector3(v0.x, v0.y, v0.z),
                     new THREE.Vector3(v1.x, v1.y, v1.z),
                     new THREE.Vector3(v2.x, v2.y, v2.z)
                 );
-                var j = geometry.vertices.length - 3;
-                geometry.faces.push(new THREE.Face3(j, j+1, j+2));
+                var j = vertices.length - 3;
+                faces.push(j, j+1, j+2);
             }
-            geometry.computeBoundingSphere();
-            geometry.computeFaceNormals();
```

Comment:
Is there a reason we don't do this anymore? Was this not necessary? These are still available on BufferGeometry.